cmd/cluster-capacity/go/src/github.com/kubernetes-incubator/cluster-capacity/Dockerfile

```diff
@@ -14,10 +14,9 @@
 
 FROM golang:latest
 
-MAINTAINER Dominika Hodovska <dhodovsk@redhat.com>
+MAINTAINER Avesh Agarwal <avagarwa@redhat.com>
 
 COPY hypercc /bin/hypercc
 RUN ln -sf /bin/hypercc /bin/cluster-capacity
 RUN ln -sf /bin/hypercc /bin/genpod
-COPY config/default-scheduler.yaml /config/default-scheduler.yaml
```

Comment:
Do we not need this?

Reply:
no

Reply:
it is used when a customized scheduler config is passed, which should be done via configmap.

Reply:
No need to hard code.

Reply:
Do we document where this configmap should be mounted inside the container?

Reply:
In https://docs.openshift.com/container-platform/3.7/admin_guide/cluster_capacity.html, we have documented how to mount pod spec via configmap but not customized scheduler config as mostly one does not need that. That said, the above documented can be updated with description about how to mount scheduler config  as a ConfigMap in a pod.